time-js/README.md

```diff
@@ -0,0 +1,11 @@
+# `time-js` module
+
+The purpose of this module is to be a smoke-test for Protobuf code generation into JavaScript.<br/>
```

Comment:
Please kill this `<br/>`.

Reply:
As discussed vocally, we need this break to make the paragraph easier to read. There's no seem to be no better way for formatting in Markdown, unless we want to have a separate paragraph. But here we don't.